src/lib/utils/os_utils.cpp

```diff
@@ -163,6 +167,27 @@ uint64_t OS::get_cpu_cycle_counter()
    return rtc;
    }
 
+long OS::get_cpu_total()
+   {
+   static long tt = 0;
+   if(tt == 0)
+#if defined(BOTAN_TARGET_OS_HAS_SYSCONF)
```

Comment:
POSIX includes sysconf but does not guarantee existence of `_SC_NPROCESSORS_xx` so this guard should be `defined(BOTAN_TARGET_OS_HAS_POSIX1) && defined(_SC_NPROCESSORS_CONF)`

I'm not sure if POSIX actually guarantees that sysconf args are macros (vs an enum) but worst case, such (weird) platforms don't get CPU count info. So whatever.